about.html

```diff
@@ -0,0 +1,164 @@
+                <li class="list-margin"><a href="#">Program</a></li>
+                <li class="list-margin"><a href="#">Join</a></li>
+                <li class="list-margin"><a href="#">Sponsor</a></li>
+                <li class="list-margin"><a href="#">News</a></li>
+                <li><a href="" id="last-menu">Crypto & New Order</a></li>
+              </ul>
+            </div>
+          </nav>
+        </div>
+      </header>
+      <section class="intro-about">
+        <div class="intro-title-about">
+          <h1>CONVECTION ABOUT CRIPTOS AND THE NEW ORDER</h1>
+        </div>
+        <div class="intro-text-about">
+          <p>
+            Our first big conference in Venezuela to talk about cryptos with the
+            best of the best! Thanks to our partner agencies around the globe,
+            we are able to make the most big conference in latin America. They
+            say they critpcurrencies are the future. They assure us they want to
+            help us become a better civilization. They teach us about the new
+            tecnologies to come. Are you ready? Sign up and prepare to the
+            future!
+          </p>
+        </div>
+        <div class="intro-contact">
+          <p>
+            Please contact us per Email for any further questions about our
+            Criptocurrencie conference!
+          </p>
+          <p class="contact-email"><strong>criptosfuture@gmail.com</strong></p>
+        </div>
+      </section>
+
+      <section class="logo-about">
+        <div class="title-logo">
+          <h2>Crypto and the New Order logo 2022</h2>
+          <div class="orange-line"></div>
+          <p>
+            The logo for the convection "CRYPTO AND THE NEW ORDER" was decided
+            from a group of the finest dessigners
+          </p>
+        </div>
+        <div class="logo-container">
+          <img src="images/logo.png" alt="Logo crypto and the new order" />
+        </div>
+      </section>
+
+      <section class="past-convention">
+        <div class="past-title">
+          <h2>See the past Crypto and New Order conventions</h2>
+          <div class="orange-line"></div>
+          <p>
+            Take a look to the last two Crypto and New Order wich took place in
+            Spain and Argentina
+          </p>
+        </div>
+        <div class="past-convention-card">
+          <div class="spain card-past">
+            <h3>2010</h3>
+            <h4>Crypto and New Order Barcelona Spain</h4>
+          </div>
+          <div class="argentina card-past">
+            <h3>2018</h3>
+            <h4>Crypto and New Order Buenos Aires Argentina</h4>
+          </div>
+        </div>
+      </section>
+
+      <section class="partner-section">
+        <div class="partner-title">
+          <h2>Partner</h2>
+          <div class="orange-line"></div>
+        </div>
+        <div class="partners-logo">
+          <img src="images/binance-logo.png" alt="binance" id="binance" />
```

Comment:
- [ ] Please remove the partners section from the about page to match the Behance layout.